### PR TITLE
Chore(1.8.2): Update docs for version 1.8.2

### PR DESCRIPTION
**Pull request by uditgaurav**
Signed-off-by: Udit Gaurav <udit.gaurav@mayadata.io>

**What this PR does / why we need it**:

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

**Checklist:**
-   [ ] Fixes #<issue number>
-   [ ] Labelled this PR & related issue with `documentation` tag

### CODE REVIEW

File: docs/node-drain.md [hunk 1]
```diff
@@ -23,7 +23,7 @@ sidebar_label: Node Drain
 ## Prerequisites
 
 - Ensure that the Litmus Chaos Operator is running by executing `kubectl get pods` in operator namespace (typically, `litmus`). If not, install from [here](https://docs.litmuschaos.io/docs/getstarted/#install-litmus)
```
Comment (divya-mohan0209) [inline docs/node-drain.md:25]: Shouldn't this be ChaosOperator, @uditgaurav @ksatchit?

File: docs/node-io-stress.md [hunk 2]
```diff
@@ -23,7 +23,7 @@ sidebar_label: Node IO Stress
 ## Prerequisites
 
 - Ensure that the Litmus Chaos Operator is running by executing `kubectl get pods` in operator namespace (typically, `litmus`). If not, install from [here](https://docs.litmuschaos.io/docs/getstarted/#install-litmus)
```
Comment (divya-mohan0209) [inline docs/node-io-stress.md:25]: Similarly for Chaos Operator, here as well? @ksatchit @uditgaurav

**Review by divya-mohan0209 — COMMENTED** [review]
LGTM except the Chaos Operator query. @ksatchit , @k8s-dev : Please have a look

File: docs/node-drain.md (hunk 1 above)
Comment (uditgaurav) [reply docs/node-drain.md:25]: Yes, @divya-mohan0209 it should be like `ChaosOperator` only. We will be fixing the formatting issues with https://github.com/litmuschaos/litmus-docs/pull/362 PR. The current one is only for the version update.

Comment (ksatchit) [reply docs/node-drain.md:25]: Will be addressed as part of #362

**Review by ksatchit — APPROVED** [review]

**Review by ispeakc0de — APPROVED** [review]

File: docs/node-io-stress.md (hunk 2 above)
Comment (ksatchit) [reply docs/node-io-stress.md:25]: Will be addressed as part of #362

**Comment by ksatchit** [comment]
Going ahead with the merge as the changes are purely related to versioning.

**Pull request merged**